htdocs/modulebuilder/template/core/triggers/interface_99_modMyModule_MyModuleTriggers.class.php

```diff
@@ -311,6 +311,9 @@ public function runTrigger($action, $object, User $user, Translate $langs, Conf
 			//case 'SHIPPING_REOPEN':
 			//case 'SHIPPING_DELETE':
 
+			// Cron jobs
+			//case 'CRONJOB_ERROR':
```

Comment:
Triggers are reserved for CRUD methods of an object, inside the update transaction.
To intercept code out of transaction context (like in your cas after running the cron_rub_jobs), you must instead use hooks.